### DISCUSSION TITLE
能支持paypal支付吗，我是中国用户，我购买不了。

### DISCUSSION THREAD

**Discussion by chunlai1688** [aclap-dev/video-downloadhelper#1326] [Help and Q&A]
### Your question

能支持paypal支付吗，我是中国用户，我购买不了。

### Addon & CoApp version and details

能支持paypal支付吗，我是中国用户，我购买不了。

**Comment by jcfrog** [comment]
Hi,
Sorry, there is no other way than using Stripe payment platform.
